GliaWidgets/Public/Glia/Glia.OpaqueAuthentication.swift

```diff
@@ -50,7 +50,10 @@ extension Glia {
         }
 
         return .init(
-            authenticateWithIdToken: { idToken, accessToken, callback in
+            authenticateWithIdToken: { [weak self] idToken, accessToken, callback in
+                self?.loggerPhase.logger.prefixed(Self.self).info(
+                    "Authenticate. Is external access token used: \(accessToken != nil)"
```

Comment:
- is it authentication result closure or it is just function. For me it looks like this log should be inside of the callback to be sure that authentication is succeeded.
- I'd propose to put the value inside of single quote to be sure that inside no spaces
```swift
 "Authenticate. Is external access token used: '\(accessToken != nil)'."
```
- `accessToken` is used for internal usage of integrator services. For debug purposes it is more useful to use `idToken` instead, since there is payload (with name, email, etc.). At the same time name, email, etc. may be sensitive data so we need to confirm that it is allowed to log.

Reply:
We log here the fact if access token is used or not, so it is boolean value. As stated [here](https://github.com/salemove/ios-sdk-widgets/pull/852#issuecomment-1854455410) proposed change has to be done on both platforms and can not be a part of this PR.